install/installer/pkg/components/ws-daemon/clusterrole.go

```diff
@@ -17,33 +18,46 @@ import (
 func clusterrole(ctx *common.RenderContext) ([]runtime.Object, error) {
 	labels := common.DefaultLabels(Component)
 
+	var rules []rbacv1.PolicyRule
+
+	_ = ctx.WithExperimental(func(cfg *experimental.Config) error {
+		if cfg.Common != nil && cfg.Common.UsePodSecurityPolicies {
+			rules = append(rules, rbacv1.PolicyRule{
+				APIGroups: []string{"policy"},
+				Resources: []string{"podsecuritypolicies"},
+				Verbs:     []string{"use"},
+				ResourceNames: []string{
+					fmt.Sprintf("%s-ns-privileged-unconfined", ctx.Namespace),
+				},
+			})
+		}
+		return nil
+	})
```

Comment:
nit: we could consider making it as a function since there are lots of components with similar code.